docs/contribute.md

```diff
@@ -112,4 +121,125 @@ $ bin/ralph fetch -b es
+
+Dependencies:
+
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/) (>`v.1.23.5`):
+  This CLI is used to communicate with the running Kubernetes instance you
+  will use.
+- [k3d](https://k3d.io/) (>`v.5.0.0`): This tool is used to setup
+  and run a lightweight Kubernetes cluster, in order to have a local
+  environment (it is required to complete below's quickstart instructions to
+  avoid depending on an existing Kubernetes cluster).
+- [curl](https://curl.se/) is required by Arnold's CLI.
+- [gnupg](https://gnupg.org/) to encrypt Ansible vaults passwords and
+  collaborate with your team.
+
+### Create a local `k3d` cluster
+
+To create (or run) a local kubernetes cluster, we use `k3d`. The cluster's
+bootstrapping should be run _via_:
+
+```bash
+$ make k3d-cluster
+```
+
+You can check that your cluster is running using the `k3d cluster` command:
+
+```bash
+$ k3d cluster list
+NAME     SERVERS   AGENTS   LOADBALANCER
+ralph    1/1       0/0      true
+```
+
+As you can see, we are running a single node cluster called `ralph`.
+
+### Bootstrap an Arnold's project
+
+Once your kubernetes cluster is running, you need to create a standard Arnold
+project describing applications and environments you need to deploy:
+
+```bash
+$ make arnold-bootstrap
+```
+
+Once bootstrapped, Arnold should have created a `group_vars` directory
+containing the following files:
+
+```bash
+$ tree group_vars
+group_vars
+├── common
+└── customer
+    └── ralph
+        ├── development
+        │   ├── main.yml
+        │   └── secrets
+        │       ├── databases.vault.yml
+        │       ├── elasticsearch.vault.yml
+        │       └── ralph.vault.yml
+        └── main.yml
+
+5 directories, 5 files
+```
+
+### Prepare working namespace
+
+You are now ready to create required Kubernetes objects to start working on
+Ralph's deployment:
+
+```bash
+$ make arnold-init
+```
+
+At this point an Elasticsearch cluster should be running on your Kubernetes
+cluster:
+
+```bash
+$ kubectl get -l app=elasticsearch pod
+NAME                                         READY   STATUS      RESTARTS   AGE
+elasticsearch-node-0                         1/1     Running     0          69s
+elasticsearch-node-1                         1/1     Running     0          69s
+elasticsearch-node-2                         1/1     Running     0          69s
+es-index-template-j-221010-09h25m24s-nx5qz   0/1     Completed   0          49s
+```
+
+We are now ready to deploy Ralph to Kubernetes!
+
+### Deploy, code, repeat
+
+To test your local docker image, you need to build it and publish it to the
+local kubernetes cluster docker registry using the `k3d-push` Makefile rule:
+
+```bash
+$ make k3d-push
+```
+
+> Note that each time you modify Ralph's application or its Docker image, you
+> will need to make this update.
+
+Now that your Docker image is published, it's time to deploy it!
+
+```bash
+$ make arnold-deploy
+```
+
```

Comment:
After the deployment, could you add a test ?
```suggestion
  seq 100 | \ 
  xargs -I {} -n 1 sh -c ' \
      echo \{\"key{}\":\"value{}\"\}' | \
  jq -s . | \
  http \
    --verify no \
    --auth username:password \
    --json \
    POST \
    https://lrs.development-ralph.192.168.1.56.nip.io/xAPI/statements/
```

Reply:
Good point! I've also added a paragraph to configure LRS authentication when using the Tray.